src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  cursor: pointer;
+}
+
+.navigation-list {
+  padding: 0;
+  list-style-type: none;
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  align-items: center;
+}
+
+.navigation {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  padding: 1.9em 0;
+}
+
+.navigation-list-item {
+  margin-right: 20px;
+}
+
+.navigation-list-item:last-child {
+  margin-right: 0;
+}
+
+.link {
+  color: #000;
+  text-decoration: underline transparent;
+  padding: 23px 0;
+}
+
+.link:hover {
+  color: #00acdc;
+  text-decoration: underline #00acdc 2px;
```

Comment:
do not use underline, just do it like in Figma design

Reply:
there is nothing about :hover on Figma or I didn't get it 

if I remove text-decoration: underline - links will lose their underline - done